migrations/server/mysql/0006_namespaces.up.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE `namespaces` (
```

Comment:
YAAAAASSSS! I was just explaining this pattern to @riyazdf and @cyli a month or so ago :-D